### DISCUSSION TITLE
Color for single select options

### DISCUSSION THREAD

**Discussion by douglaslassance** [community/community#6234] [Projects and Issues] [answered]
Not necessarily vital, but still nice, would be the ability to set a color on each option of a single select field.

**Comment by Katsute** [comment]
You could use emojis as one way of color coding them ⚫⚪🔴🔵🟠🟡🟢🟣🟤

![gh_projects](https://user-images.githubusercontent.com/58778985/136983519-b8726b22-84f0-4a4a-9911-fc4cc9855852.PNG)

**Comment by ynouri** [comment]
Emojis help but having a custom color background (exactly like in existing, regular Github issues labels) is even more helpful. The background color makes it very easy for the eyes and brain to identify what issues belongs to what single select category.

In our case, we would like to use this for Epics

**Comment by debkanchan** [comment]
Trying to switch from notion this is posing a challenge as I could see the color and instantly know what's the status of a task.

**Comment by archon810** [comment]
If we can already assign colors to labels, we should be able to assign colors to any custom fields in Projects Beta. Functionally, they're pretty much the same.

**Comment by j0siepy** [comment, marked as answer]
Hi folks!! Thank you so much for your patience - we've made this possible! Let us know if you have any future feedback for us ❤️ 

https://user-images.githubusercontent.com/98360703/217907438-c0d2e60a-95e5-4d61-a5b2-7af2a9d0a5d1.mov

**Reply by archon810** [reply]
Thank you!

Can we have more colors, or maybe add custom colors?

Also, wrong place to ask, but maybe you know - is reordering of custom fields being worked on at all? They're stuck in the order they were created.

**Reply by uo9qsuf** [reply]
Thank very much, but can we have more colors? 7 is not enough. One color picker will solve all problems.

**Reply by j0siepy** [reply]
In order to ensure accessibility for folks with color blindness - at the moment, we can only support 7 colours. 

> Also, wrong place to ask, but maybe you know - is reordering of custom fields being worked on at all? They're stuck in the order they were created.

If you go to settings, you can reorder custom single select fields. 

<img width="1440" alt="Screenshot 2023-02-13 at 10 33 34 AM" src="https://user-images.githubusercontent.com/98360703/218501505-0a83dc53-36f8-4f56-8f0b-05fa20e2c658.png">

**Reply by archon810** [reply]
> If you go to settings, you can reorder custom single select fields.

I think we have a little misunderstanding here. I'm trying to reorder the actual custom field order, not the items within each custom field.

These (in our case):
![image](https://user-images.githubusercontent.com/469543/218675143-b60e5384-43a3-4190-b458-130b7a6c1fb2.png)

There's no way to reorder them AFAIK. For example, the sprint field will always follow the custom tag field, and there's no way to move them around.

**Reply by uo9qsuf** [reply]
> In order to ensure accessibility for folks with color blindness - at the moment, we can only support 7 colours.
> 
> > Also, wrong place to ask, but maybe you know - is reordering of custom fields being worked on at all? They're stuck in the order they were created.
> 
> If you go to settings, you can reorder custom single select fields.
> 
> <img alt="Screenshot 2023-02-13 at 10 33 34 AM" width="1440" src="https://user-images.githubusercontent.com/98360703/218501505-0a83dc53-36f8-4f56-8f0b-05fa20e2c658.png">

Thank you, the accessibility reason is understandable, but should you get this a so high priority? The labels have a text and so people with color blindness use as a fallback. But all other have a hard limit, and 7 colors a really not so useful.
